src/main/java/seedu/address/logic/parser/DisplayCommandParser.java

```diff
@@ -20,14 +23,62 @@
     public DisplayCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args);
+        List<String> argList = argMultimap.getAllValues(EMPTY_PREFIX);
+        String lastArgument = extractLastArgument(argList);
 
         Index index;
         try {
+            if (hasParameterValue(argList)) {
```

Comment:
It doesnt make sense that you try to extractLastArgument(argList) before checking if hasParameterValue(argList)

Reply:
hmm yea ur right, let me change that